<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialogClass</class>
 <widget class="QDialog" name="settingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>316</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>504</width>
    <height>316</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>504</width>
    <height>316</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="optionsList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>12</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>298</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>298</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QStackedWidget" name="pagesWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="videoOptions">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="enableShadersCheck">
         <property name="text">
          <string>Enable Shaders (only recommended for some games with NVIDIA card)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="soundOptions">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="SoundBufferSizeLabel">
         <property name="text">
          <string>Sound buffer count:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QSlider" name="SoundBufferSizeSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="muteAudioCheckBox">
         <property name="text">
          <string>Mute audio</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="SoundBufferSizeSpinBox">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="suffix">
          <string> X frames</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="SoundBufferDefaultSizeButton">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="memcardOptions">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="disableMemStickCheck">
         <property name="text">
          <string>Disable Memcard insert ( no memstick inserted behaviour)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hackOptions">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="disableAudioFucCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Disable some audio functions known to freeze
some games (e.g. gripshift) as a temporary hack.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="decryptorcheckBox">
         <property name="text">
          <string>Enable Decryptor (experimental)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="SaveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pcsp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>optionsList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>settingsDialogClass</receiver>
   <slot>changePage(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveButton</sender>
   <signal>pressed()</signal>
   <receiver>settingsDialogClass</receiver>
   <slot>SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>pressed()</signal>
   <receiver>settingsDialogClass</receiver>
   <slot>SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SoundBufferDefaultSizeButton</sender>
   <signal>pressed()</signal>
   <receiver>settingsDialogClass</receiver>
   <slot>soundBufferDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SoundBufferSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SoundBufferSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SoundBufferSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SoundBufferSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changePage(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>SaveAction()</slot>
  <slot>CancelAction()</slot>
  <slot>soundBufferDefault()</slot>
 </slots>
</ui>
